fusor-ember-cli/app/templates/components/review-link.hbs

```diff
@@ -6,6 +6,14 @@
   <div class='col-sm-6 col-md-8 col-lg-9'>
     {{#if isExternalURL}}
         <a href={{value}} target="_blank">{{value}}</a>
+        {{#if ipAddress}}
+          <button {{action 'doNothing' bubbles=false}}
+                  class="pficon fa fa-info-circle rhci-popover-button"
+                  data-toggle="popover"
+                  data-content='{{helpTextIpAddress}}'>
```

Comment:
@cfchase, @eriknelson, FYI, if I put `content` inside the JS `.popver()` call on `didInsertElement()` then it was `undefined` but if it's here in the template, it is populated after the computed property resolves.